Module Name/manifest.php

```diff
@@ -20,42 +20,94 @@
 // This file describes the module, including database tables
 
 // Basic variables
-$name        = '';            // The name of the module as it appears to users. Needs to be unique to installation. Also the name of the folder that holds the unit.
-$description = '';            // Short text description
+$name        = 'Deep Learning';            // The name of the module as it appears to users. Needs to be unique to installation. Also the name of the folder that holds the unit.
+$description = 'filler';            // Short text description
 $entryURL    = "index.php";   // The landing page for the unit, used in the main menu
 $type        = "Additional";  // Do not change.
 $category    = '';            // The main menu area to place the module in
-$version     = '';            // Version number
-$author      = '';            // Your name
+$version     = '0.1';            // Version number
+$author      = 'Harry Merrett';            // Your name
 $url         = '';            // Your URL
 
 // Module tables & gibbonSettings entries
-$moduleTables[] = ''; // One array entry for every database table you need to create. Might be nice to preface the table name with the module name, to keep the db neat. 
-$moduleTables[] = ''; // Also can be used to put data into gibbonSettings. Other sql can be run, but resulting data will not be cleaned up on uninstall.
+$moduleTables[] = 'CREATE TABLE `deepLearningEvent` (
+  `deepLearningEventID` int(10) unsigned zerofill NOT NULL AUTO_INCREMENT,
+  `name` varchar(40),
+  `description` text,
+  `backgroundImage` text,
+  `active` boolean,
+  PRIMARY KEY (`deepLearningEventID`)
+)'; // One array entry for every database table you need to create. Might be nice to preface the table name with the module name, to keep the db neat.
+$moduleTables[] = 'CREATE TABLE `deepLearningExperience` (
+  `deepLearningExperienceID` int(10) unsigned zerofill NOT NULL AUTO_INCREMENT,
+  `deepLearningEventID` int(10) unsigned,
+  `deepLearningMajorID1` int(10) unsigned
+  `deepLearningMajorID2` int(10) unsigned
```

Comment:
Looks like your manifest is off to a good start 👍  I wonder if you may have created these table definitions by hand and not tested them, because they appear to have some syntax errors (missing commas) and a few other things missing (NULL/NOT NULL declarations). Perhaps try creating these tables in a tool like Sequel Ace and then exporting them.